packages/redux-routine/src/runtime.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import { create } from 'rungen';
+import { map, isString } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import castError from './cast-error';
+
+/**
+ * Create a co-routine runtime.
+ *
+ * @param {Object}    controls Object of control handlers.
+ * @param {function}  dispatch Unhandled action dispatch.
```

Comment:
We should settle what this should be at some point. My previous comment is nonsense because `typeof {} === 'object'` (lowercase), yet we still use `Object` (uppercase) to describe the type.

Either to be discussed in Core JS or at least search through prior discussion to find what basis we had settled on here.

I'm thinking "Function" is correct though.